bin.src/processFile.py

```diff
@@ -323,12 +328,13 @@ def __iter__(self):
                         help="display the specified information to stdout and quit (unless run is specified).")
     parser.add_argument("-L", "--loglevel", help="logging level", default="WARN",
                         choices=CaseInsensitiveChoices('DEBUG', 'INFO', 'WARN', 'FATAL'))
-    parser.add_argument("-T", "--trace", nargs="*", action=pbArgparse.TraceLevelAction,
-                        help="trace level for component", metavar="COMPONENT=LEVEL")
+    if False:
```

Comment:
Add a comment explaining
(a) why the `--trace` option is currently disable
(b) when it should be reactivated